users/kuchosauronad0/kuchosauronad0.h

```diff
@@ -1,8 +1,7 @@
+#pragma once
 #ifndef USERSPACE
```

Comment:
Unless your are checking this is defined somewhere, `#pragma once` replaces these guards
```suggestion
```